src/core/assert_json.rs

```diff
@@ -1,8 +1,47 @@
+use crate::prelude::*;
+
 use assert_json_diff::assert_json_include;
 use core::fmt::Debug;
 use pretty_assertions::{assert_eq, assert_ne};
-use serde::{de::DeserializeOwned, ser::Serialize};
+use serde::de::DeserializeOwned;
 use serde_json::Value;
+use thiserror::Error as ThisError;
+
+#[derive(Debug, ThisError)]
+pub enum TestingError {
```

Comment:
moved from `vector` integration tests, now used by that integration test but also by GW model unit tests :)